aws/policy/security-services.yaml

```diff
@@ -106,6 +106,7 @@ Statement:
       - secretsmanager:PutResourcePolicy
       - secretsmanager:DeleteResourcePolicy
       - secretsmanager:RemoveRegionsFromReplication
+      - secretsmanager:GetRandomPassword
```

Comment:
I think the problem is that this policy statement is resource restricted and the `GetRandomPassword` permission does not operate on a secret. I just tested by adding this permission to the `AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees` policy statement and the associated PR passes. Unfortunately, this policy is right at the maximum allowed characters and there's not a lot here that can be collapsed. About the only thing I can see would be to replace `kms:DisableKey` and `kms:DisableKeyRotation` with `kms:Disable*`. There are only two `kms:Disable*` permissions so this won't add anything new.

Reply:
Sounds good. I've reduced the two rules to `kms:Disable*`.